rbac/server/api/roles.py

```diff
@@ -80,6 +84,34 @@
         await utils.send(
             request.app.config.VAL_CONN, batch_list, request.app.config.TIMEOUT
         )
+        conn.close()
+        if role_title != "NextAdmins":
+            distinguished_name_formatted = "CN=" + role_title + "," + GROUP_BASE_DN
+            data_formatted = {
+                "created_date": r.now(),
+                "distinguished_name": distinguished_name_formatted,
+                "group_nickname": role_title,
+                "group_types": -2147483646,
```

Comment:
What happens if we don't pass in group_types field? Because this value might vary for each LDAP instance I'm not 100% sure.

Reply:
As we discussed it appears to be associated with Security Groups.